susemanager-utils/susemanager-sls/salt/images/docker.sls

```diff
@@ -35,6 +35,18 @@ mgr_pushimage:
       - module: mgr_buildimage
       - module: mgr_registries_login
 
+{% if grains['saltversioninfo'][0] >= 2019 %}
```

Comment:
I'm not sure this would work as we want. The problem here is that we have to consider that minions might be not up-to-date, so they could have a Salt 2019.2.0 version installed where the `docker.logout` method is not yet available.

Maybe something like this would be better:
```jinja
{% if 'docker.logout' in salt %}
...
{% endif %}
```

Reply:
Right, minions could be on 2019.2.0 but not on the MU that adds `docker.logout`. I'll try your suggestion.